about.css

```diff
@@ -284,37 +284,45 @@ body {
 .sponsor-title {
   margin-top: 10%;
   text-align: center;
+  color: #fff;
 }
 
 .sponsor-img {
   margin-top: 5%;
-  width: 25rem;
+  width: 5rem;
   list-style-type: none;
+  opacity: 100%;
+}
+
+.div-sponsor {
+  background-color: #2e2e2e;
 }
 
 .sponsor {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
+  flex-wrap: wrap;
   align-items: center;
   padding: 5%;
-  gap: 8rem;
+  gap: 1rem;
+  margin-left: 10%;
```

Comment:
- [ ] (OPTIONAL) Kindly stick to the original design as much as possible. As a result, it would be better to style this section like the logo section 👍 
- Your design:
![Screenshot 2022-07-02 054907](https://user-images.githubusercontent.com/85113567/176986962-45e6b45d-60dd-4db5-aba9-15b615ebcccc.png)
- Required design:
![Screenshot 2022-07-02 054924](https://user-images.githubusercontent.com/85113567/176986970-04270f8f-d6c4-427c-844e-3bc386b17635.png)